docker/scripts/scripts/httpsender/Alert_on_Unexpected_Content_Types.js

```diff
@@ -11,6 +11,7 @@ var expectedTypes = [
 		"application/json",
 		"application/octet-stream",
 		"application/problem+json",
+		"application/health+json",
```

Comment:
Could you move to the beginning of the array to keep them in alphabetic order?

Reply:
Sure